css-properties-values-api/Overview.bs

```diff
@@ -711,21 +711,10 @@ Conditional Rules {#conditional-rules}
 --------------------------------------
 
 <div class=note>
-	''@supports'' rules and the {{CSS/supports(conditionText)}} method
+	''@supports'' rules and the {{CSS/supports}} method
```

Comment:
s/method/methods?

Reply:
"method" is fine (there's only one of them! overloads are a WebIDL fiction!), but it should be `{{CSS/supports()}}`.